docs/documentation/maya/simshape.md

```diff
@@ -38,90 +37,124 @@ In order to add or remove any of those optional meshes, a set of menu items are
+ - **Global Mass Multiplier** (Float, 1.0): Sets the scaling factor applied to the mass of every point.
+    - Has a range of \[0.0, 10.0\] [^1]
+ - **Global Damping** (Float, 0.75): Sets the scaling factor applied to the global damping of every point.
+    - Has a range of \[0.0, 1.0\] [^1]
+ - **Inertia Damper** (Float, 0.0): Sets the linear damping applied to the dynamics of every point.
+    - Has a range of \[0.0, 1.0\] [^1]
+ - **Rest Length Multiplier** (Float, 1.0): Sets the scaling factor applied to the edge lengths at rest.
+    - Has a range of \[0.0, 2.0\] [^1]
+ - **Compression Multiplier** (Float, 1.0): Sets the scaling factor applied to the compression resistance of every point.
+    - Has a range of \[0.0, 2.0\] [^1]
+ - **Stretching Multiplier** (Float, 1.0): Sets the scaling factor applied to the stretching resistance of every point.
+    - Has a range of \[0.0, 2.0\] [^1]
+ - **Attenuation Velocity factor** (Float, 1.0): Sets the weight of the attenuation applied to the whole simulation driven by the Attenuation Matrix.
+    - Has a range of \[0.0, 10.0\] [^1]
 
 #### Collision Settings
-| Attribute            | Type  | Value | Range/Options      | Description                  |
-| :------------        | :---  | :---- | :------------      | :--------------------------- |
-| Compute Collisions   | Bool  | True  | \[False, True\]    | Flag to enable collisions correction. |
-| Keep Orientation     | Bool  | True  | \[False, True\]    | Preserves the initial orientation of the vertices relative to the collider when handling collisions.|
-| Max Sliding Distance | Float | 1.0   | \[0.0, 10.0\] [^1] | Maximum distance the simulated vertex is allowed to slide on top of the collider in world units. |
+ - **Compute Collisions** (Boolean, True): Flag to enable collisions correction in the deformer. If disabled, the deformer will ignore colliders when deforming the mesh.
+ - **Keep Orientation** (Boolean, True): Flag to preserve the initial orientation of the vertices relative to the collider when handling collisions. If disabled, the mesh will suffer no changes if the orientation of the collider varies.
+ - **Max Sliding Distance** (Float, 1.0): Maximum distance (in world units) the simulated vertex is allowed to slide relative to the collider.
+    - Has a range of \[0.0, 10.0\] [^1]
 
 #### Attraction Settings
-| Attribute             | Type | Value     | Range/Options  | Description           |
-| :------------         | :--- | :----     | :------------  | :----------           |
-| Attraction Remap Mode | Enum | Cube Root | <ul><li>Linear</li><li>Logarithmic</li><li>Square Root</li><li>Cube Root</li></ul> | Remap mode used to compute the definitive attraction values. |
+ - **Attraction Multiplier** (Float, 1.0): Maximum distance (in world units) the simulated vertex is allowed to slide relative to the collider.
```

Comment:
I think this definition corresponds to Max Sliding Distance instead of Attraction Multiplier.

Reply:
My bad, yes! The definition should be:
```suggestion
 - **Attraction Multiplier** (Float, 1.0): Sets the scaling factor applied to the Attraction.
```

Also, the range should be corrected for it to be a soft max of 2.0